genum/definitions.go

```diff
@@ -33,8 +33,4 @@ type TypedEnum[T EnumLike] interface {
 
 	// Values returns all valid values of an enum.
 	Values() []T
-
-	// ParseString converts text into a type if valid.
-	// returns true if the enum is valid, and false otherwise.
-	ParseString(text string) (T, error)
```

Comment:
After thinking about this some more, I want to put this back- it can be Parse(any) and delegate to the static parser, but I like it attached to the type for the way it works with generics; i.e. I don't want to pass a type and a parser, when I can get the parser from the type.

Reply:
ok sure. its reverted